pythran/pythonic/bisect/bisect_impl.hpp

```diff
@@ -0,0 +1,25 @@
+#ifndef PYTHONIC_BISECT_BISECT_IMPL_HPP
+#define PYTHONIC_BISECT_BISECT_IMPL_HPP
+
+#include "pythonic/utils/proxy.hpp"
+#include "pythonic/bisect/bisect.hpp"
+
+#include <algorithm>
+
+namespace pythonic {
+
+    namespace bisect {
+        template <class X, class A>
+            size_t bisect(X const& x, A const& a, long lo, long hi) {
+                if(hi<0) hi = x.end() - x.begin();
+                return std::upper_bound(x.begin()+lo, x.begin()+hi, a) - x.begin();
+            }
+
+        PROXY_IMPL(pythonic::bisect, bisect);
+
+    }
+
+}
+
+#endif
+
```

Comment:
Yes but it will required to generate correct inclusion like:

```
#include <lala.h>
#include <lolo.h>
#include <lala_impl.h>
#include <lolo_impl.h>
```
in the Pythran generated file but it can't be done until all the xxxx_impl.hpp file are created. So for now, I include them here and once all of the xxx_impl.hpp files exists, I will generated correct include directives.
I can do all of this in a same PR but it means no modification can be done until the end of this PR...